warmup.py

```diff
@@ -225,7 +227,12 @@ class RadialDistribution:
 
     """
     def __init__(self, rmax, dr, rmin=0.0):
-        pass
+        self.rmax = rmax
+        self.dr = (rmax - rmin) / np.round((rmax - rmin) / dr)
```

Comment:
This is correct, but for readability, you might want to first compute the denominator (which is the number of bins) in a separate variable so that the code is easier to understand later.